azure-toolkit-libs/azure-toolkit-sqlserver-lib/src/main/java/com/microsoft/azure/toolkit/lib/sqlserver/model/SqlServerEntity.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.sqlserver.model;
+
+import com.microsoft.azure.toolkit.lib.common.entity.IAzureResourceEntity;
+import com.microsoft.azure.toolkit.lib.common.model.Region;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder(toBuilder = true)
+public class SqlServerEntity implements IAzureResourceEntity {
+
+    private String name;
+    private String id;
+    private String resourceGroup;
+    private String subscriptionId;
+    private Region region;
+
+    private String kind;
+    private String administratorLogin;
```

Comment:
shall we rename it to `administratorLoginName`? as `administratorLogin` is a bit confused